saskatoon/harvest/admin.py

```diff
@@ -66,6 +66,39 @@ class PropertyImageInline(admin.TabularInline):
 class PropertyAdmin(LeafletGeoAdmin):
     model = Property
     inlines = [PropertyImageInline]
+    list_display = (
+        "__str__",
+        "authorized",
+        "pending",
+        "harvest_every_year", 
+        "approximative_maturity_date",
+        "avg_nb_required_pickers",
+        "neighborhood",
+        "city"
+    )
+    list_filter = (
+        "authorized",
+        "pending",
+        "harvest_every_year", 
+        "trees_location",
+        "avg_nb_required_pickers",
+        "public_access",
+        "neighbor_access",
+        "compost_bin",
+        "ladder_available",
+        "ladder_available_for_outside_picks",
+        "trees",
+        "postal_code",
+        "neighborhood",
+        "city"
+    )
+    search_fields = (
+        "street_number",
+        "street",
+        "complement",
+        "publishable_location",
+        "additional_info",
```

Comment:
I don't think these are relevant. I would rather be able to search info about the owner, e.g. `owner__person__last_name` or `owner__person__authuser__email`